.github/pull_request_template.md

```diff
@@ -28,6 +28,7 @@ Hey! You're PRing? Cool! Please have a look at the below checklist. It's here to
     - [ ] Link new and existing constants in docstrings instead of hard coded number and strings
     - [ ] Add new message types to `Message.effective_attachment`
     - [ ] Added new handlers for new update types
+      - [ ] Add the handlers to the warning loop in the `ConversationHandler`
```

Comment:
my intention was to indent it, because it is only needed if new handlers are added. Apparently I need to do two spaces, so thats wrong here anyway, but I would still indent it.

Reply:
ah okay. good point 👍